File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        s = s.toLowerCase();
        boolean isFirst = true;
        StringBuilder sb = new StringBuilder();
        
       for(char c : s.toCharArray()) {
           if(c==' ') {
               sb.append(' ');
               isFirst = true;
           }
           else{
               if(isFirst) {
                   sb.append(Character.toUpperCase(c));
                   isFirst = false;
               } else {
                   sb.append(c);
               }
           }
        }
        return sb.toString();
      
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    s = s.toLowerCase()
    let words = s.split(' ')
    let arr = words.map((word)=>{
        return word ? word[0].toUpperCase() + word.slice(1).toLowerCase() : ''
    })
    return arr.join(' ');
}